css/src/_m-message.scss

```diff
@@ -0,0 +1,72 @@
+.m-messages {
+    padding: 0;
+    font-style: italic;
+    list-style: none;
+
+    &__item {
+        --message-txt: #da394c;
+        --message-bg: rgba(238, 162, 171, 0.25);
+        position: relative;
+        width: fit-content;
+        margin-inline-end: 11px;
+        margin-block-end: 20px;
+        padding: 10px 15px;
+
+        color: var(--message-txt);
+        background-color: var(--message-bg);
+        border-radius: 8px;
+
+        &--warning {
+            --message-txt: #c18d0a;
+            --message-bg: rgba(248, 218, 141, 0.25);
+        }
+
+        &--info {
+            --message-txt: #4f7c8e;
+            --message-bg: rgba(181, 207, 218, 0.25);
+        }
+
+        &--success {
+            --message-txt: #659e57;
+            --message-bg: rgba(194, 227, 186, 0.25);
+        }
+    }
+
+    &__close-btn {
+        position: absolute;
+        top: -11px;
+        right: -11px;
+    }
+}
+
+@media (prefers-color-scheme: dark) {
```

Comment:
Move this block inside the existing `.m-messages` block like shown below,

```scss
.m-messages {
    /* ... */

    @media (prefers-color-scheme: dark) {
        &__item {
            /* ... */
        }
    }
}

Reply:
Moved the @media block inside the existing `.m-messages` block like the above shown structure.